associations/has_many_association.go

```diff
@@ -119,3 +122,52 @@ func (a *hasManyAssociation) AfterSetup() error {
 	}
 	return nil
 }
+
+func (a *hasManyAssociation) AfterProcess() AssociationStatement {
+	v := a.value
+	if v.Kind() == reflect.Ptr {
+		v = v.Elem()
+	}
+
+	otherIDField := "ID"
+
+	ownerIDField := "ID"
+	ownerID := reflect.Indirect(reflect.ValueOf(a.owner)).FieldByName(ownerIDField).Interface()
+
+	ids := []interface{}{ownerID}
+
+	for i := 0; i < v.Len(); i++ {
+		fval := v.Index(i).FieldByName(otherIDField)
+
+		id := ""
+		if fval.Type().Name() == "UUID" {
+			id = fval.Interface().(uuid.UUID).String()
+		} else {
+			id = fmt.Sprint(fval.Interface())
+		}
+		if id != "0" && id != emptyUUID {
+			ids = append(ids, id)
+		}
+	}
+	if len(ids) <= 1 {
+		return AssociationStatement{
+			Statement: "",
+			Args:      []interface{}{},
+		}
+	}
+
+	fk := a.fkID
+	if fk == "" {
+		fk = inflect.Underscore(a.ownerName) + "_id"
+	}
+
+	ret := fmt.Sprintf("UPDATE %s SET %s = ? WHERE %s in (?", a.tableName, fk, otherIDField)
```

Comment:
I don't understand what's the aim of the `IN` here. Can you explain how it works? :)

Reply:
updating all the associated models to point to this model's ID.

Reply:
Can you add comments on how it works, then? (from a user point of view) Also maybe you can use sqlx.In here instead of constructing the IN tokens yourself.